src/service/FlowerStore.java

```diff
@@ -0,0 +1,50 @@
+package service;
+
+import com.flower.kiev.*;
```

Comment:
Don't use a wild card in imports.  I have already written about it in chat. You could tune IntelliJ idea not to use .* in imports.